Makefile

```diff
@@ -929,9 +929,8 @@ test_m65: build/mega65.rom
 testremote: build/kernal_custom.rom build/basic_custom.rom $(TARGET_CHR_PXL) build/symbols_custom.vs
 	x64 -kernal build/kernal_custom.rom -basic build/basic_custom.rom -chargen $(TARGET_CHR_PXL) -moncommands build/symbols_custom.vs -remotemonitor
 
-testsimilarity: $(TOOL_SIMILARITY) $(DIR_GEN)/OUTx_x.BIN $(ROM_CBM_KERNAL) $(ROM_CBM_BASIC)
+testsimilarity: $(TOOL_SIMILARITY) $(DIR_GEN)/OUTx_x.BIN $(ROM_CBM_KERNAL)
 	$(TOOL_SIMILARITY) $(ROM_CBM_KERNAL) $(DIR_GEN)/OUTx_x.BIN
-	$(TOOL_SIMILARITY) $(ROM_CBM_BASIC)  $(DIR_GEN)/OUTx_x.BIN
```

Comment:
I didn't mean completely removing the BASIC ROM similarity checks - Commodore might have made some improvements there, we still need the check.

What I meant was to be able to skip checking individual procedures, like $BC0C-$BC1A area. My proposal: mask this area with 0's, either using some script here (so that the similarity tool would receive an altered ROM images), or within the similarity tool itself.